src/components/Contributions.js

```diff
@@ -26,7 +26,7 @@ function Contributions({repoName, owner, viewer}) {
       .catch(e => {
         console.log(e);
       });
-  }, []);
+  }, [owner, repoName, viewer]);
```

Comment:
Curios what was happening here?

Reply:
All of these are basically https://reactjs.org/docs/hooks-rules.html and https://stackoverflow.com/a/58959298/757375